cli/cmd/csv_render.go

```diff
@@ -0,0 +1,54 @@
+//
+// Author:: Matt Cadorette (<matthew.cadorette@lacework.net>)
+// Copyright:: Copyright 2020, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cmd
+
+import (
+	"encoding/csv"
+	"fmt"
+	"strings"
+)
+
+// Used to clean CSV inputs prior to rendering
+func csvCleanData(input []string) []string {
+	var data []string
+	for _, h := range input {
+		data = append(data, strings.Replace(h, "\n", "", -1))
+	}
+	return data
+}
+
+// Used to produce CSV output
+func renderAsCSV(headers []string, data [][]string) string {
```

Comment:
I think this would be better placed in outputs.go alongside the OutputJson function.

`func (c *cliState) OutputCSV(v interface{})
`

As the CSV writer only operates on []string we could use 	"github.com/fatih/structs"

something to the likes of this

```
	for _, header := range structs.Fields(v) {
		headers = append(headers, header.Name())
	}
	
	for _, value := range structs.Values(v) {
		values = append(values, value.(string))
	}
```